src/utils/hash.ts

```diff
@@ -1,7 +1,11 @@
 import { createHash } from 'crypto';
 import { Readable } from 'stream';
 
-import { Sha256Prefix } from '../common.js';
+/** 1220 is the starting prefix for all sha256 multihashes
+ *  0x12 - ID of sha256 multi hash
+ *  0x20 - 32 bytes (256 bits) of data
+ */
```

Comment:
It might be useful to link to [the Multihash standard](https://multiformats.io/multihash/) here.